dotcom-rendering/src/components/AdSlot.apps.stories.tsx

```diff
@@ -1,21 +1,30 @@
+import { ArticleDesign, ArticleDisplay, Pillar } from '@guardian/libs';
 import { breakpoints } from '@guardian/source-foundations';
 import { useRef } from 'react';
+import { splitTheme } from '../../.storybook/decorators/splitThemeDecorator';
 import { AdSlot, type Props } from './AdSlot.apps';
 
 export default {
 	component: AdSlot,
 	title: 'Components/AdSlot.apps',
 	parameters: {
 		chromatic: {
-			viewports: [breakpoints.mobile, breakpoints.tablet],
+			viewports: [breakpoints.phablet, breakpoints.tablet],
```

Comment:
Why did we change this viewport, rather than adding to it?

Reply:
I think it might be worth [having a version of this decorator that splits vertically, as mentioned in the original PR](https://github.com/guardian/dotcom-rendering/pull/9269#pullrequestreview-1692307393). Not a blocker, but something we might want to tackle sooner rather than later.